noobaa/noobaa-core
# fix #5791 - add missing extensions to generated ssl certificates

### Explain the changes
1. Reduce certificate expiry to 2 years
2. Add to generated SSL certificates the following extensions as required (see issue #5791 for details):
```
ExtendedKeyUsage=serverAuth
SubjectAlternativeName=DNS:selfsigned.noobaa.io
```

### Issues: Fixed #xxx / Gap #xxx
1. Fixed #5791

### Testing Instructions:
1. Use updated OS, for example MacOS Catalina
2. Install a new noobaa system
3. Open the frontend using https address and make sure you get `NET::ERR_CERT_AUTHORITY_INVALID` which can be skipped manually, and not `NET::ERR_CERT_REVOKED` or `NET::ERR_CERT_INVALID`.
